### DISCUSSION TITLE
[Documentation] Restoring backups

### DISCUSSION THREAD

**Discussion by elliotpryde** [mbround18/valheim-docker#144] [Help!] [answered]
There's currently no explanation of how to restore a backup once it's been taken. I've tried a few different things, but moving the sole `saves` directory within the backup archive to `/home/steam/.config/unity3d/IronGate/Valheim` doesn't seem to have any effect on my character. Any explanation on how this works would be appreciated.

**Comment by mbround18** [comment, marked as answer]
Are you trying to restore your world? or character?

**Reply by elliotpryde** [reply]
Both if possible.

**Reply by mbround18** [reply]
Character files are stored locally and this repo could not possibly have any control over your local files :P 

As for the world file, it dumps backups into `/home/steam/backups`. To restore a backup file:
1. `gosu steam bash`
2. `cd /home/steam/backups`
3. `tar -zxf your-backup-name.tar.gz --directory /home/steam/tmp`
4. `cd /home/steam/tmp/saves`
5. `mv . /home/steam/.config/unity3d/IronGate/Valheim`
6. Restart your server.

**Reply by elliotpryde** [reply]
Ahh, that makes sense since characters can move between servers 🤦 

Thanks for the world backup restore instructions 👍
